template.yaml

```diff
@@ -136,8 +136,13 @@ Parameters:
       - "warn"
       - "error"
       - "debug"
+  PermissionBoundaryPolicyArn:
+    Description: ARN of a boundary policy if your organisation uses some for roles, optional.
+    Type: String
+    Default: "true"
```

Comment:
Other params in this template use default ""
Let's use that instead of "true" ok?